lib/erb_lint/linters/rubocop.rb

```diff
@@ -82,7 +82,8 @@ def inspect_content(processed_source, erb_node)
         team = build_team
         team.inspect_file(source)
         team.cops.each do |cop|
-          cop.offenses.select(&:corrected?).each_with_index do |rubocop_offense, index|
+          cop.offenses.each_with_index do |rubocop_offense, index|
```

Comment:
I think you will need to `partition` the offenses on `&:corrected?` and treat each set of offenses separately. Otherwise the corrections may end up being mis-aligned (wrong correction applied for an offense). The reason is that corrections are pushed by rubocop into `cop.corrections` only when the offense is corrected, so you may have more elements in `cop.offenses` than in `cop.corrections`. If you `each_with_index` on `cop.offenses` without first elimitating the uncorrected offenses, `cop.corrections[index]` won't align properly.

Reply:
Ah yeah got it, thanks for the explanation. Will try to create a test to reproduce the issue